### ISSUE TITLE
Modularize the pixelManipulation API

### ISSUE THREAD

**Issue by tech4GT** [publiclab/image-sequencer#550]
I think we should break out parts like export image from pixel manipulation, it should make the whole api a little more flexible.
cc @jywarren

**Comment by Mridul97** [comment]
I would love to work on this. Can I take up this issue?

**Comment by tech4GT** [comment]
Mridul let's collaborate on this, i was thinking maybe we can push this on top of the other modularization issue.
We need to be careful with this though, not to break any existing code since a lot of it depends on pixemanipulation.

**Comment by Mridul97** [comment]
Okay, that sounds good!

**Comment by jywarren** [comment]
Yes, and perhaps some of the expanded per-module tests I opened an issue
for could help us to build confidence as we further modularize code.

For this one, i think we might develop a folder in, say,
/src/pixel-manipulation/ or /src/util/pixel-manipulation/ (or another that
makes sense to you!) to contain subparts of the pixelManipulation code. How
does that sound?

**Comment by Mridul97** [comment]
@Tech4GT in the Pixel Manipulation file, there are 4 functions used changePixel, extraManipulation, getPixels, savePixels. I did not get exportImage part in Pixel Manipulation file Could you please elaborate a little more.. Thanks!
